src/data/matrix/basic.lean

```diff
@@ -878,6 +882,13 @@ by { ext, refl }
 
 end transpose
 
+section has_star
+variables [has_star α]
+instance : has_star (matrix n n α) := {star := conj_transpose}
+lemma conj_transpose_eq_star_of_square_matrix (M : matrix m m α) :
+Mᴴ = star M := rfl
```

Comment:
This is now the third time I'm telling you this, but:
```suggestion
lemma star_eq_conj_transpose (M : matrix m m α) : star M = Mᴴ := rfl
```

Please try not to apply suggestions then revert them.

Reply:
Sorry Eric, I confused myself when changing things.

Reply:
This comment still applies - your version still has the old (and now backwards) lemma name.

Reply:
But I have already changed that. I looked again, Can you have a double-check?

Reply:
OK, thanks! Sorry for troubling you. I thought the change is only the eq direction.